src/questions.js

```diff
@@ -0,0 +1,99 @@
+// Importing dependancies
+const inquirer = require("inquirer");
+const fs = require("fs");
+
+// Initial Manager and team name Questions
+const introQuestions = [
+	{
+		type: "input",
+		message: "Please enter your team name",
+		name: "teamName",
+		validate: (answer) => (!answer ? "please enter a name" : true),
+	},
+
+	{
+		type: "input",
+		message: "Please enter managers name:",
+		name: "name",
+		filter: (answer) => answer.toUpperCase(),
+		validate: (answer) => (!answer ? "please enter a name" : true),
+	},
+	{
+		type: "input",
+		message: "Please enter id:",
+		name: "id",
+		validate: (answer) =>
+			isNaN(answer) ? "Your ID must be only numbers - please try again" : true,
+	},
+	{
+		type: "input",
+		message: "Please enter email:",
+		name: "email",
+		validate: (answer) => (!answer ? "please enter an email" : true),
```

Comment:
Try to see if you can check if the inputted value is actually a valid email, rather than checking for any input. You could use a regex for this.